services/galley/src/Galley/Cassandra/Instances.hs

```diff
@@ -56,14 +56,12 @@ instance Cql ConvType where
   toCql SelfConv = CqlInt 1
   toCql One2OneConv = CqlInt 2
   toCql ConnectConv = CqlInt 3
-  toCql GlobalTeamConv = CqlInt 4
 
   fromCql (CqlInt i) = case i of
     0 -> pure RegularConv
     1 -> pure SelfConv
     2 -> pure One2OneConv
     3 -> pure ConnectConv
-    4 -> pure GlobalTeamConv
```

Comment:
I think we need to keep this. There are currently already values in the DB with "4". If we don't keep this then processes will crash upon reading